examples/jsm/controls/ArcballControls.js

```diff
@@ -0,0 +1,2495 @@
+import {
+	GridHelper,
+	EllipseCurve,
+	BufferGeometry,
+	Line,
+	LineBasicMaterial,
+	Raycaster,
+	Group,
+	Object3D,
+	Box3,
+	Sphere,
+	Quaternion,
+	Vector2,
+	Vector3,
+	Matrix4,
+	MathUtils
+} from '../../../build/three.module.js';
+import {} from 'https://cdnjs.cloudflare.com/ajax/libs/hammer.js/2.0.8/hammer.js'
```

Comment:
Is it possible to avoid the usage of `hammer.js`?

In any event, external libraries should not be included like that. If you really need it, you have to copy it into the `examples/jsm/libs` folder.

Reply:
I choosed `hammer.js` for recognizing multi-touch gestures for touch interaction and actually is my main recognizer for pointer events, that's why i would prefer to keep using it.

Reply:
In other controls like `OrbitControls` we detect multi-touch gestures without a library. I have the feeling `hammer.js` could be a blocker for this PR.

Reply:
Indeed! Would be great if `ArcballControls` didn't have any dependencies 👍

As @Mugen87 said, you can use `OrbitControls` as reference:

https://github.com/mrdoob/three.js/blob/2f6358dd55956e4f3595f304b2b2c925551228d3/examples/jsm/controls/OrbitControls.js#L800-L874

Reply:
Hi, I am @danielefornari advisor on this project.
The problem here is having high level cross-browser support for touch gestures and mouse interaction and, as far as I know, right now, there is no clear way to do it easily without an external library. `hammer.js` is a temporary solution, and I would stick with it while we figure out what to do next.
In any case, for high level interaction, I think it makes a lot of sense to delegate the handling to an external library. Does anyone knows a better/more up-to-date lib for this purpose?

Reply:
> The problem here is having high level cross-browser support for touch gestures and mouse interaction and, as far as I know, right now, there is no clear way to do it easily without an external library.

Yes there is. [Pointer Events API](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events) unifies mouse events and touch events in a single API.